test/cri-containerd/main.go

```diff
@@ -4,29 +4,51 @@ package cri_containerd
 
 import (
 	"context"
+	"fmt"
 	"net"
 	"testing"
 	"time"
 
 	_ "github.com/Microsoft/hcsshim/test/functional/manifest"
+	registry "golang.org/x/sys/windows/registry"
 	"google.golang.org/grpc"
 	runtime "k8s.io/kubernetes/pkg/kubelet/apis/cri/runtime/v1alpha2"
 )
 
 const (
-	daemonAddress                = "tcp://127.0.0.1:2376"
-	connectTimeout               = time.Second * 10
-	testNamespace                = "cri-containerd-test"
-	wcowProcessRuntimeHandler    = "runhcs-wcow-process"
-	wcowHypervisorRuntimeHandler = "runhcs-wcow-hypervisor"
-	lcowRuntimeHandler           = "runhcs-lcow"
-	imageWindowsRS5Nanoserver    = "mcr.microsoft.com/windows/nanoserver:1809"
-	imageWindowsRS5Servercore    = "mcr.microsoft.com/windows/servercore:1809"
-	imageLcowK8sPause            = "k8s.gcr.io/pause:3.1"
-	imageLcowAlpine              = "docker.io/library/alpine:latest"
-	imageLcowCosmos              = "cosmosarno/spark-master:2.4.1_2019-04-18_8e864ce"
+	daemonAddress                     = "tcp://127.0.0.1:2376"
+	connectTimeout                    = time.Second * 10
+	testNamespace                     = "cri-containerd-test"
+	wcowProcessRuntimeHandler         = "runhcs-wcow-process"
+	wcowHypervisorRuntimeHandler      = "runhcs-wcow-hypervisor"
+	wcowHypervisor17763RuntimeHandler = "runhcs-wcow-hypervisor-17763"
+	lcowRuntimeHandler                = "runhcs-lcow"
+	imageLcowK8sPause                 = "k8s.gcr.io/pause:3.1"
+	imageLcowAlpine                   = "docker.io/library/alpine:latest"
+	imageLcowCosmos                   = "cosmosarno/spark-master:2.4.1_2019-04-18_8e864ce"
 )
 
+var (
+	relID                       = getHostReleaseID()
+	imageWindowsNanoserver      = "mcr.microsoft.com/windows/nanoserver:" + relID
+	imageWindowsServercore      = "mcr.microsoft.com/windows/servercore:" + relID
+	imageWindowsNanoserver17763 = "mcr.microsoft.com/windows/nanoserver:1809"
```

Comment:
Maybe we should make this a func for Windows?

`getWindowsNanoserverImage()` -> default for this OS ver
`getWindowsNanoserverImage(build int)` -> matching one.
And usage would be `getWindowsNanoserverImage(osversion.RS5)`

And then variants for `getWindowsServercoreImage(), getWindowsServercoreImage(build int)`

Reply:
Then instead of relID you can just do osversion.Get().Build by default.

Reply:
yeah, that looks more clean. I will get this